src/Components/Article_card/Article_card.jsx

```diff
@@ -1,5 +1,78 @@
-import React from "react";
+import React, { useEffect, usearticle, useState } from "react";
+import { useParams, Link } from "react-router-dom";
+import { getArticles } from "../../utils/API";
+import {
+  Wrapper,
+  Content,
+  Title,
+  Button,
+  AuthorTopic,
+  Votes,
+} from "./Article_card.styles";
 
-export default function Article_card() {
-  return <div>Article_card</div>;
-}
+const Article_card = () => {
```

Comment:
I think there may have been some confusion as to what an ArticleCard component should be doing. An article card would be a single card(in this case containing the information about a single article). 

Right now, your Article_card component is rendering all of the articles(which is fine if you want it to do so), but I just wanted to highlight that semantically, a card would not be used for this purpose. 

What you are more likely to see, is an Articles components, that would subsequently render a selection of ArticleCards. I am happy to discuss this over zoom if there is any confusion.

Reply:
Updated folder and function names